python/llm/test/run-llm-inference-tests-gpu-mistral.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
rename to run-llm-inference-tests-gpu-434.sh? may have other models so it is not good to add mistral in the name. use 434 to indicate the transformers version?